### PR TITLE
Removes Fun

### PR DESCRIPTION
**Pull request by TheDZD**

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
fun is gone. All gone.

Damn it DZD, why you remove fun.

File: html/changelog.html [hunk 1]
```diff
@@ -55,6 +55,42 @@
...
+				<li class="tweak">Changes vampire blood display to be similar to ling chemical counter</li>
+				<li class="bugfix">Fixes unnecessary toxin damage being dealt on severe bloodloss</li>
+				<li class="bugfix">Fixes human mobs not receiving proper random names</li>
+				<li class="rscadd">Adds in knight armor</li>
+				<li class="rscadd">Grants the chaplain a set of crusader knight armor</li>
+				<li class="rscadd">Adds in cult-resistant ERT paranormal space suit</li>
+				<li class="rscadd">Allows the Chaplain to convert his null rod into a holy sword with crusader knight armor</li>
+				<li class="tweak">Chaplain's closet is now a secure closet</li>
+				<li class="rscadd">Adds in Assault Gear crate to cargo</li>
+				<li class="rscadd">Adds in military assault belt rcsadd: Adds in spaceworthy swat suits</li>
```
Comment (Fox-McCloud) [inline html/changelog.html]: something went wrong here.

Comment (TheDZD) [inline html/changelog.html]: Huh, that's odd. Something went wrong during the automatic changelog generation. Let me fix that.

**Comment by NTSAM** [comment]
I thought you already removed fun?

**Comment by TheDZD** [comment]
@NTSAM There is always more fun to remove.

**Comment by NTSAM** [comment]
tru

**Comment by KasparoVy** [comment]
RIP

**Comment by Tastyfish** [comment]
Did @tigercat2000 intentionally call themself tigetbat2000 in the log?

**Comment by tigercat2000** [comment]
maybe i did maybe i didn't
you'll never get my secrets

**Comment by Tastyfish** [comment]
It is an enigma.

**Pull request merged**